server/views/taskList/taskList.pug

```diff
@@ -34,15 +34,17 @@ block content
                     include caSubmitAddressReviewTask
 
             else if (['DECIDED', 'MODIFIED', 'MODIFIED_APPROVAL'].includes(licenceStatus.stage))
-                include eligibilitySummaryTask
+                if (licenceStatus.decisions.curfewAddressApproved !== 'withdrawn')
+                    include eligibilitySummaryTask
```

Comment:
Is it right to hide the eligibility if address withdrawn? I haven't seen the reqs but seems odd. Not that odd is odd for licences...